python/lsst/afw/geom/linearTransform.cc

```diff
@@ -57,6 +57,20 @@ PYBIND11_PLUGIN(_linearTransform) {
     clsLinearTransform.def(py::init<typename LinearTransform::Matrix const &>());
 
     /* Operators */
+    clsLinearTransform.def("__mul__", [](LinearTransform const & self,
+                                         LinearTransform const & other) {
+            return self*other;
+        }, py::is_operator());
+    // __getitem__ implemented in linearTransform.py
+    clsLinearTransform.def("_getitem_nochecking", [](LinearTransform const & self,
+                                                     int i) {
+            return self[i];
```

Comment:
Not yet. The main advantage over using the 1d function on each axis separately is a better error message. It might be worth adding for that reason. If you and @kfindeisen want it now for use on this ticket I can whip it together (or feel free).